apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -0,0 +1,301 @@
+          )
+          mainContent = (
+            <Prose>
+              <h1>USB Drive Detected!</h1>
+              <p>
+                <USBImage src="usb-drive.svg" alt="Insert USB Image" />
+                Would you like to export the ballot configuration now? A zip
+                archive will automatically be saved to the inserted USB drive.
+                You may also {/* eslint-disable jsx-a11y/anchor-is-valid */}
+                <a href="#" onClick={openFileDialog} data-testid="manual-link">
+                  manually select a location to save the archive to.
+                </a>
+                {/* eslint-enable jsx-a11y/anchor-is-valid */}
+              </p>
+            </Prose>
+          )
+          break
+        }
+      }
+      break
+
+    case 'RenderBallot': {
+      disableCancel = true
+      const {
+        ballotStyleId,
+        precinctId,
+        contestIds,
+        isLiveMode,
+        locales,
+      } = state.currentBallotConfig
+      const precinctName = getPrecinctById({ election, precinctId })!.name
+
+      mainContent = (
+        <Prose>
+          <h1>
+            Generating Ballot{' '}
+            {state.ballotConfigsCount - state.remainingBallotConfigs.length} of{' '}
+            {state.ballotConfigsCount}…
+          </h1>
+          <ul>
+            <li>
+              Ballot Style: <strong>{ballotStyleId}</strong>
+            </li>
+            <li>
+              Precinct: <strong>{precinctName}</strong>
+            </li>
+            <li>
+              Contest count: <strong>{contestIds.length}</strong>
+            </li>
+            <li>
+              Language format:{' '}
+              <strong>{getHumanBallotLanguageFormat(locales)}</strong>
+            </li>
+            <li>
+              Filename:{' '}
+              <Monospace>{state.currentBallotConfig.filename}</Monospace>
+            </li>
+          </ul>
+          <HandMarkedPaperBallot
+            ballotStyleId={ballotStyleId}
+            election={election}
+            electionHash={electionHash}
+            isLiveMode={isLiveMode}
+            precinctId={precinctId}
+            onRendered={onRendered}
+            locales={locales}
+          />
+        </Prose>
+      )
+      break
+    }
+
+    case 'ArchiveEnd': {
+      mainContent = (
+        <Prose>
+          <h1>Finishing Download…</h1>
+          <p>
+            Rendered {pluralize('ballot', state.ballotConfigsCount, true)},
+            closing zip file.
+          </p>
+        </Prose>
+      )
+      break
+    }
+
+    case 'Done': {
+      primaryButton = <USBControllerButton primary small={false} />
+      mainContent = (
+        <Prose>
+          <h1>Download Complete!</h1>
+          <p>
+            Exported {pluralize('ballot', state.ballotConfigsCount, true)}. You
+            may now eject the USB device and connect it with your ballot
+            scanning machine to configure it.
+          </p>
+        </Prose>
```

Comment:
<img width="611" alt="image" src="https://user-images.githubusercontent.com/1938/100913004-0b9c8080-3486-11eb-8e31-0d10fa2e4596.png">

This could be less wordy. Or even replaced with a graphic showing the USB stick moving to the BSD? @beausmith

Reply:
My suggestion in code is above.